tmt/schemas/common.yaml

```diff
@@ -493,15 +493,3 @@ definitions:
     type: string
     # yamllint disable-line rule:line-length
     pattern: "^\\d{2,}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}\\.\\d+\\+\\d{2}:\\d{2}$"
-
```

Comment:
Does it make sense to move it to test.yaml, or is check expected to be used elsewhere in future?

Reply:
I am hoping we can move the triggering of check to the plan, with the expectation that the check will operate on each test in the plan. But I am not sure if that work has been green lighted yet.